test/unittests/mongoBackend/mongoUpdateContextCompoundValues_test.cpp

```diff
@@ -141,19 +141,21 @@
                                                                          \
     x    = str->add(orion::ValueTypeVector, "x", "");                    \
     leaf = str->add(orion::ValueTypeNumber, "y", 3.0);                   \
+    leaf = str->add(orion::ValueTypeNone, "z", "");                      \
```

Comment:
I have realized that `leaf` variable is not really neeced, so before merging this PR I propose to remove it (to make the test macro more clear). What do you think?